packages/trip-form/src/SettingsSelectorPanel/index.js

```diff
@@ -93,9 +93,12 @@ export default class SettingsSelectorPanel extends Component {
       // If there are multiple (scooter | bikeshare) providers,
       // then if one is specified by the mode button, select it,
       // othewise select all providers.
+      // selectedCompanies is at least an empty array.
       const selectedCompanies =
         defaultCompany ||
-        getCompanies(supportedCompanies, nonTransitModes).map(comp => comp.id);
+        (getCompanies(supportedCompanies, nonTransitModes) || []).map(
```

Comment:
Why not just change the default props value for supportedCompanies to an empty array?

Reply:
supportedCompanies is passed from config.companies, which can be null/undefined.

Reply:
But that seems to work too!

Reply:
Yea, my thinking here is that either `companies` should be defined in the config with an array OR if it's missing then it would default to this empty array. I don't think we necessarily need to tolerate the case where it is null. So anywhere you check that it exists `companies && companies.length > ...` you may be able to just remove that first check.